tests/udf/test_simple_udf.py

```diff
@@ -770,3 +772,70 @@ def test_previous_attrs(lt_ctx, cls):
     # the current alternative
     with pytest.raises(AttributeError, match='self.meta.'):
         lt_ctx.run_udf(dataset=ds, udf=cls())
+
+
+class BadGetMethodUDF(UDF):
+    def get_result_buffers(self):
+        return {}
+
+    def get_method(self):
+        return self.params.get('method', 42)
+
+
+@pytest.mark.parametrize(
+    'method',
+    [
+        42,
+        UDFMethod.FRAME,
+        UDFMethod.PARTITION,
+        UDFMethod.TILE,
+    ]
+)
+def test_bad_get_method(lt_ctx, method):
+    ds = lt_ctx.load('memory', data=np.ones((2, 2, 4, 4)))
+    with pytest.raises(UDFException):
```

Comment:
I was a bit confused by this test case, maybe adding a comment here helps:

```suggestion
    # `UDFException` is thrown either for the invalid method `42`, or
    # because the UDF doesn't actually implement the matching `process_*` methods:
    with pytest.raises(UDFException):
```